examples/src/main/java/ai/timefold/solver/examples/meetingscheduling/score/MeetingSchedulingConstraintProvider.java

```diff
@@ -51,7 +51,7 @@ protected Constraint roomConflict(ConstraintFactory constraintFactory) {
     }
 
     protected Constraint avoidOvertime(ConstraintFactory constraintFactory) {
-        return constraintFactory.forEachIncludingNullVars(MeetingAssignment.class)
+        return constraintFactory.forEachIncludingUnassigned(MeetingAssignment.class)
```

Comment:
I think `MeetingAssignment` still uses `nullable=true`, since I don't see it in the changed files.

Reply:
`MeetingAssignment` never used `nullable = true`.
This code is there for construction heuristics, because this problem has 2 genuine variables.